scripts/automated_ingestion/eessitarball.py

```diff
@@ -72,13 +72,14 @@ def find_state(self):
             try:
                 self.git_repo.get_contents(state + '/' + self.metadata_file)
                 return state
-            except github.UnknownObjectException:
```

Comment:
Don't we want to keep this, in case PyGithub is updated to map `GithubException` with status `404` to `UnknownObjectException`?

Reply:
Good idea, done in https://github.com/EESSI/filesystem-layer/pull/206/commits/2a04cf1af6e40de478ca8db90d81649df8e63c34.